README.md

```diff
@@ -114,6 +114,16 @@ NVD: **CWE-269: Improper Privilege Management**
 Same problem as CWE-2022-23116 in a different component.
 
 
+## [CVE-2022-25180](https://nvd.nist.gov/vuln/detail/CVE-2022-25180)
+
+### CVE Assignents
+
+Jenkins Project: **CWE-522: Insufficiently Protected Credentials**  
+NVD: **CWE-319: Cleartext Transmission of Sensitive Information**
+
+The password is never transmitted in plain text, that's not the problem.
```

Comment:
Perhaps we should briefly mention what the problem is here to have higher chance of convincing them? If we care enough of course.

Reply:
Unsure. If they've read the CVE description and advisory, it's unclear to me how they got to 319, so more explanation seems unnecessary for now.